core/unix/signal_linux_arm.c

```diff
@@ -109,8 +130,38 @@ signal_frame_extra_size(bool include_alignment)
     return 0;
 }
 
+static void
+vfp_query_signal_handler(int sig, siginfo_t *siginfo, kernel_ucontext_t *ucxt)
+{
+    uint offset = 160; /* size of IWMMXT frame */
```

Comment:
I've defined the struct in sigcontext.h (though we're not very interested in it).